### PR TITLE
Finished SBA 1.5.6

### PR DESCRIPTION
**Pull request by boiscljo**
I finished the code that was left as a WIP. Updated the libraries so it works on 1.18 servers and tested with BW 0.2.21. Unfortunately I disabled the party system of SBA as BW 0.2.21 has already support for it.

### CODE REVIEW

File: .github/workflows/gradle-publish.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - name: Set the Build number
+        id: calculated_build_number
+        env:
+          NUM: ${{ github.run_number }}
+        run: echo ::set-output name=build_n::$(($NUM+865))
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```
Comment (zlataovce) [inline .github/workflows/gradle-publish.yml:21]: `adopt` has been renamed to `temurin`

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,6 @@
 */build/
 gradle-app.setting
 !gradle-wrapper.jar
-.gradletasknamecache
\ No newline at end of file
+.gradletasknamecache
+*/bin/*
+*.class
```
Comment (zlataovce) [inline .gitignore:12]: missing `.settings\`, `.classpath` and `.project`

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```
Comment (zlataovce) [inline gradle/wrapper/gradle-wrapper.properties:3]: 7.3.1 is the latest

File: gradle.properties [hunk 4]
```diff
@@ -1,8 +1,8 @@
-SBA_VERSION=1.5.6-SNAPSHOT
+SBA_VERSION=1.5.6-dev29
```
Comment (zlataovce) [inline gradle.properties:1]: change this to 1.5.6

File: plugin/src/main/java/io/github/pronze/sba/SBA.java [hunk 5]
```diff
@@ -60,14 +60,15 @@
         id = "SBA",
         authors = {"pronze"},
         loadTime = Plugin.LoadTime.POSTWORLD,
-        version = "1.5.6-SNAPSHOT"
+        version = "1.5.6-dev29"
```
Comment (zlataovce) [inline plugin/src/main/java/io/github/pronze/sba/SBA.java:63]: change this to 1.5.6

**Review by zlataovce — APPROVED** [review]

**Comment by boiscljo** [comment]
Branch v1 had commit made after the Pull Request that were not meant to be merged with the original repository, I'll open a new pull request on a clean branch without any .setting and applying all @zlataovce suggestions.

**Pull request closed without merging**